asf_heat_pump_suitability/pipeline/run_scripts/run_add_features.py

```diff
@@ -169,16 +166,18 @@ def parse_arguments() -> argparse.Namespace:
     )
 
     logging.info("Adding grid capacity column to EPC")
-    grid_capacities = grid_capacity.calculate_grid_capacity()
+    grid_capacities = grid_capacity.calculate_grid_capacity().select(
+        ["lsoa", "heatpump_installation_percentage"]
+    )
     epc_df = epc_df.join(grid_capacities, how="left", on="lsoa")
```

Comment:
I think the nulls in `heat_pump_installation_percentage` and `has_anchor_property` are true nulls. Meaning I think we are missing the data to say what their HP installation percentage is and whether or not they have anchor properties. Therefore I think they should remain as nulls rather than filling with False.

@shabrf is this correct?

Reply:
ah I see!

Reply:
For heat pump installations the null values would be true nulls as these appear when we don't have geo data for the substation distribution area which covers the LSOA, so the capacity available to that LSOA is unknown. For anchor properties, we can assume the the points of interest dataset has good coverage so null values here would indicate that the LSOA doesn't contain any of the specified anchor properties within it, so I think we should fill these nulls with False.